lib/rules/selector-max-compound-selectors/__tests__/index.mjs

```diff
@@ -384,3 +384,116 @@ testRule({
 		},
 	],
 });
+
+// Testing ignoreSelectors
+testRule({
+	ruleName,
+	config: [2, { ignoreSelectors: ['::v-deep', '/-moz-.*/', /-screen$/, '.ignored'] }],
+
+	accept: [
+		{
+			code: '::v-deep .foo .bar {}',
+			description: 'ignored selector at the beginning',
+		},
+		{
+			code: '.foo ::v-deep .bar {}',
+			description: 'ignored selector in the middle',
+		},
+		{
+			code: '.foo ::v-deep > .bar {}',
+			description: 'ignored selector in the middle followed by a combinator',
+		},
+		{
+			code: '.foo::v-deep .bar {}',
+			description: 'ignored selector as pseudo-element',
+		},
+		{
+			code: '.foo::v-deep > .bar {}',
+			description: 'ignored selector as pseudo-element followed by a combinator',
+		},
+		{
+			code: '.foo > ::v-deep .bar {}',
+			description: 'ignored selector preceded by a combinator',
+		},
+		{
+			code: '.foo input::-moz-placeholder {}',
+			description: 'selector ignored by string regex',
+		},
+		{
+			code: '.foo :-webkit-full-screen a {}',
```

Comment:
nit
non-blocking

You could refactor by removing `/-screen$/` and just depending on `/-moz-.*/`.
 The reason being that both `::-moz-placeholder`  *and* `:-moz-placeholder` exist.
i.e. `.foo:-moz-placeholder a {}`

Reply:
I think this case checks a regex-like string (`'/-moz-.*/'`) and a regex (`/-screen$/`), so keeping it as-is makes sense for me.